spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/SparkTestHelperBase.java

```diff
@@ -55,12 +55,15 @@ private Object[] toJava(Row row) {
 
   protected void assertEquals(
       String context, List<Object[]> expectedRows, List<Object[]> actualRows) {
-    Assert.assertEquals(
-        context + ": number of results should match", expectedRows.size(), actualRows.size());
+    Assertions.assertThat(actualRows.size())
```

Comment:
it's better to have
```
Assertions.assertThat(actualRows)
        .as(context + ": number of results should match")
        .hasSameSizeAs(expectedRows);
```
because this will show the content when the assertion ever fails

Reply:
Confirmed that on my end, thanks. Update the tests.